packages/igx-templates/igx-ts/custom-templates/crm-grid/files/src/app/__path__/__filePrefix__.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="grid__wrapper">
-	<igx-grid #grid1 [data]="localData" [height]="'430px'" [width]="'1180px'" [rowSelection]="'multiple'"
+	<igx-grid #grid1  igxPreventDocumentScroll id="grid1" [data]="localData" [height]="'430px'" [width]="'1180px'" [rowSelection]="selectionMode"
```

Comment:
Changing the rowSelection in this case does not hurt, however - it is a bit obsolete as we are just setting it to 'multiple' in the .ts file. So, it is up to you to decide if the change is really needed.

Reply:
Changed to the suggested.